lms/djangoapps/courseware/transformers.py

```diff
@@ -0,0 +1,93 @@
+"""
```

Comment:
Hmm.. I'd like to keep our python modules relatively small, if possible, going forward.  So rather than having a single file with multiple transformers, I'd prefer having each transformer in their own file.
Alternatively, you can create a new folder within `courseware` called `transformers` and put `grades.py` in it.